proposals/0016-c-interface-compiler-library.md

```diff
@@ -492,11 +830,100 @@ if (errors && errors->GetStringLength() > 0) {
 ```
 
 ### Optimizing a shader
+A shader can be optimized by providing different passes lists to the optimizer.
```

Comment:
Would be great to have some insight into how widely this is used.  I think it is quite telling that Copilot is unable to write me any sensible example code that uses this interface.

Reply:
Agreed.  It was hard to find info on this to even write my snippet.  Nothing really out there about it.